mino/minogrpc/rpc.go

```diff
@@ -143,8 +144,16 @@ func (rpc RPC) Stream(ctx context.Context, players mino.Players) (mino.Sender, m
 
 	md := metadata.Pairs(
 		headerURIKey, rpc.uri,
-		headerStreamIDKey, streamID,
-		headerGatewayKey, rpc.overlay.myAddrStr)
+		HeaderStreamIDKey, streamID,
+		headerGatewayKey, rpc.overlay.myAddrStr,
+	)
+
+	protocol := ctx.Value(tracing.ProtocolKey)
+	if protocol != nil {
+		md.Append(tracing.ProtocolTag, protocol.(string))
+	} else {
+		md.Append(tracing.ProtocolTag, "undefined")
```

Comment:
this "undefined" looks like a magic value, which could actually match a protocol's name.

I would suggest creating a constant with an unlikely protocol name, such as `__UNDEFINED_PROTOCOL__` or similar